test/python/taskbasedsimulator/benchmark.py

```diff
@@ -28,124 +29,139 @@ def execute_circuit(qc: QuantumCircuit, backend, shots: int, mode: str | ddsim.P
           sep=';')
 
 
-def execute_verification(qc: QuantumCircuit, backend, shots: int,
+def execute_verification(qc: QuantumCircuit, qcog: QuantumCircuit, gatecost, backend, shots: int,
                          mode: str | ddsim.PathSimulatorMode = 'sequential', **options):
     print('Starting execution of circuit', qc.name)
     configuration_dict = backend.configuration().to_dict()
     basis_gates = configuration_dict['basis_gates']
     print('Transpiling circuit')
     qc = transpile(qc, basis_gates=basis_gates, optimization_level=0)
-    # Compose G G'^{-1}
-    qcinv = qc.inverse()
-    qccomp = qc.compose(qcinv)
```

Comment:
Maybe I am just not seeing the point, but why is this not considering the verification scenario any more?

Reply:
The composed circuit is build in `run_benchmark` and from there passed on to `execute_verification_all`.

Reply:
Ah, thanks. Missed that. Then please go ahead, resolve and merge whenever you feel like it.

Reply:
Thanks

Reply:
Thanks